docs/en/03_handling-requests.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Handling Requests
```

Comment:
I can't think of a clean way to change this title to more clearly indicate this is for static caching (so it's more obvious in search results)... if you think of one please change it, otherwise I'll have to live with it lol